diagnostic.md

```diff
@@ -177,5 +194,5 @@ with the class `.stretch` be at a _screen width_ of 550px?
 ```
 
 <!-- your answer starts here -->
-
+100px
```

Comment:
The correct answer here would be 137.5px. Can you see why that would be the case?